src/Kernel/client/ExecutionPathVisualizer/styles.ts

```diff
@@ -109,6 +109,9 @@ export const style = (customStyle: StyleConfig = {}): string => {
     .oplus {
         fill: ${styleConfig.oplus};
     }
+    [data-zoom-in="true"] {
```

Comment:
Out of curiosity, what motivates the use of data- attributes over CSS classes for marking zoomable elements?

Reply:
No particular reason other than that the current implementation of the visualizer allows configurability through data attributes. So, instead of also allowing to pass in extra classes to add to the element, I chose to just use the already exposed configurability. Classes might've worked a little better in this case since it was just a boolean value but I felt that it was similar enough that it didn't warrant that extra change to the API of the visualizer. Open for suggestions though!

Reply:
Either way definitely works, just was a little surprising to see an attribute selector instead of a class selector, since the latter are generally a bit easier to work with. That said, I figured there was likely a good reason such that I didn't want to flag the use of attribute selectors as a problem per se, so much as to understand. Thanks for the explanation!